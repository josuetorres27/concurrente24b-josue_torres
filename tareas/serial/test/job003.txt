plate003.bin 30 18.8 250 2
plate003.bin 30 18.8 250 0.2
plate003.bin 30 18.8 250 0.02
plate003.bin 30 18.8 250 0.002
plate003.bin 30 18.8 250 0.0002
plate003.bin 30 18.8 250 0.00002
plate003.bin 30 18.8 250 0.000002
plate003.bin 30 18.8 250 0.0000002
plate003.bin 30 18.8 250 0.00000002
plate003.bin 30 18.8 250 0.000000002
plate003.bin 30 18.8 250 0.0000000002
plate003.bin 30 18.8 250 0.00000000002
plate003.bin 30 18.8 250 0.000000000002
plate003.bin 30 18.8 250 0.0000000000002
plate003.bin 30 0.34 250 0.2
plate003.bin 30 0.34 250 0.02
plate003.bin 30 0.34 250 0.002
plate003.bin 30 0.34 250 0.0002
plate003.bin 30 0.34 250 0.00002
plate003.bin 30 0.34 250 0.000002
plate003.bin 30 0.34 250 0.0000002
plate003.bin 30 0.34 250 0.00000002
plate003.bin 30 0.34 250 0.000000002
plate003.bin 30 0.34 250 0.0000000002
plate003.bin 30 0.34 250 0.00000000002
plate003.bin 30 0.34 250 0.000000000002
plate003.bin 30 0.34 250 0.0000000000002
