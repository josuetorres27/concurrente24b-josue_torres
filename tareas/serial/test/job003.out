plate003.bin	30	18.8	250	2	8	0000/00/00	00:04:00
plate003.bin	30	18.8	250	0.2	150	0000/00/00	01:15:00
plate003.bin	30	18.8	250	0.02	716	0000/00/00	05:58:00
plate003.bin	30	18.8	250	0.002	1515	0000/00/00	12:37:30
plate003.bin	30	18.8	250	0.0002	3120	0000/00/01	02:00:00
plate003.bin	30	18.8	250	2e-05	5472	0000/00/01	21:36:00
plate003.bin	30	18.8	250	2e-06	7988	0000/00/02	18:34:00
plate003.bin	30	18.8	250	2e-07	10508	0000/00/03	15:34:00
plate003.bin	30	18.8	250	2e-08	13027	0000/00/04	12:33:30
plate003.bin	30	18.8	250	2e-09	15548	0000/00/05	09:34:00
plate003.bin	30	18.8	250	2e-10	18072	0000/00/06	06:36:00
plate003.bin	30	18.8	250	2e-11	20601	0000/00/07	03:40:30
plate003.bin	30	18.8	250	2e-12	23136	0000/00/08	00:48:00
plate003.bin	30	18.8	250	2e-13	25667	0000/00/08	21:53:30
plate003.bin	30	0.34	250	0.2	1	0000/00/00	00:00:30
plate003.bin	30	0.34	250	0.02	1548	0000/00/00	12:54:00
plate003.bin	30	0.34	250	0.002	14287	0000/00/04	23:03:30
plate003.bin	30	0.34	250	0.0002	49772	0000/00/17	06:46:00
plate003.bin	30	0.34	250	2e-05	99035	0000/01/03	09:17:30
plate003.bin	30	0.34	250	2e-06	204083	0000/02/11	20:41:30
plate003.bin	30	0.34	250	2e-07	338234	0000/03/27	10:37:00
plate003.bin	30	0.34	250	2e-08	477708	0000/05/14	20:54:00
plate003.bin	30	0.34	250	2e-09	617100	0000/07/02	06:30:00
plate003.bin	30	0.34	250	2e-10	756437	0000/08/19	15:38:30
plate003.bin	30	0.34	250	2e-11	895918	0000/10/07	01:59:00
plate003.bin	30	0.34	250	2e-12	1035677	0000/11/25	14:38:30
plate003.bin	30	0.34	250	2e-13	1175264	0001/01/12	01:52:00
